test/gpu/native/deviceAttributes.prediff

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 
-./deviceAttributes --runBaseline=true > deviceAttributes.good
+NL=""
+if [[ "$CHPL_COMM" != "none" && -n "$CHPL_COMM" ]]; then
+  NL="-nl 1"
+fi
+
+./deviceAttributes $NL --runBaseline=true > deviceAttributes.good
```

Comment:
Note that you can just pass `-nl 1` to a program compiled with `COMM=none` and it is ignored, so you can simplify this and just always pass `-nl 1`